akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1250,7 +1259,7 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
 
   def receiveGetReplicaCount(): Unit = {
     // selfAddress is not included in the set
-    sender() ! ReplicaCount(nodes.size + 1)
+    sender() ! ReplicaCount(nodes.size + weaklyUpNodes.size + 1)
```

Comment:
perhaps we should include two counts in the `ReplicaCount`, depending on what we decide for the direct replication